### PR TITLE
[refactor] Refactor all factories - changes to controllers are only fixed service names

### PR DESCRIPTION
**Pull request by RRDuke**

### CODE REVIEW

File: modules/core/client/controllers/navigation.controller.client.js [hunk 1]
```diff
@@ -28,9 +28,9 @@ function NavigationController ($state, Auth, Nav, User) {
    * Initial set-up of navigation tabs
    */
   function activate () {
-    vm.tabs = _.filter(Nav.tabs, function (tab) {
-    // intersection returns empty array when no intersection is found
-    // thus we can use this to determine whether or not a tab should be visible
+    vm.tabs = _.filter(navService.tabs, function (tab) {
+    //intersection returns empty array when no intersection is found
+    //thus we can use this to determine whether or not a tab should be visible
```
Comment (codydaig) [inline modules/core/client/controllers/navigation.controller.client.js]: Why did the space get deleted from these comments?

File: modules/core/client/services/auth.service.client.js [hunk 2]
```diff
@@ -1,66 +1,82 @@
...
+  //////////
 
-    return auth;
+  function checkAuth () {
+    return $http({
+      method: 'POST',
+      url: 'api/core/users/checkauth'
+    })
+    .then(userLogin)
+    .catch(handleAuthError);
```
Comment (codydaig) [inline modules/core/client/services/auth.service.client.js:30]: I don't like this inconsistency on spacing. .then should be indented.

To make it look pretty, move the object to it's own variable:

```
var req = {
  method: 'POST',
  url: 'api/core/users/checkauth'
};

return $http(req)
  .then(userLogin)
  .catch(handleAuthError);
```

File: modules/core/client/services/user.service.client.js [hunk 3]
```diff
@@ -1,26 +1,33 @@
...
-      },
-      logIn: function (newUser) {
-        user.currentUser = newUser;
-        user.isLoggedIn = true;
-      }
-    };
-    return user;
-  });
+      login: login,
+      setDefault: setDefault,
```
Comment (codydaig) [inline modules/core/client/services/user.service.client.js]: Trailing Comma!!

**Comment by marcoviappiani** [comment]
LGTM

**Pull request merged**